metabase/metabase
# 4. Connect illustrations to new setting values

Epic: #39246

### Description

Connect the rest 3 illustration settings to the UI
1. Landing page (existing)
1. No question results
1. No search results

### How to verify
Change the 4 illustration settings to any values and visit
1. Login page
2. Landing page
3. Unsubscribe page `/unsubscribe?hash=hash&email=email&pulse-id=pulse-id`
4. Question without results
5. Questions without results as a card in a dashboard
6. `+ New` > Dashboard > Click the collection > you'll see the new collection picker
7. From the collection picker search anything that would return nothing
8. From the collection picker go to an empty collection
9. search in the app for anything that would return nothing

### Demo
![Screenshot 2024-03-14 at 10 45 39 PM](https://github.com/metabase/metabase/assets/1937582/9623d7e2-c4a6-4d24-822d-bd74e0872742)


### Checklist
